settings_schema.json

```diff
@@ -0,0 +1,61 @@
+{
+    "type" : "object",
+    "properties" : {
+        "oneVizionUrl" : {
+            "type" : "string",
+            "minLength" : 1
+        },
+        "oneVizionLogin" : {
+            "type" : "string",
+            "minLength" : 1
+        },
+        "oneVizionPwd" : {
+            "type" : "string",
+            "minLength" : 1
+        },
+        "serviceId" : {
+            "type" : "integer",
+            "minimum" : 1000000,
```

Comment:
How it can be minimal value for ID? service_id maybe equal to 1.
![image](https://user-images.githubusercontent.com/13415820/179495771-50ea4a0f-307b-4b63-88b8-6b538b80f71d.png)

Reply:
Fixed